text/0123-separators-settings-api.md

```diff
@@ -0,0 +1,164 @@
+# Separators Setting API
+
+## 1. Summary
+
+This specification describes the `separatorTokens` and `nonSeparatorTokens` index setting API endpoints.
+
+## 2. Motivation
+N/A
+
+## 3. Functional Specification
+
+### 3.1. Explanations
+
+TBD
+
+#### 3.1.1. Usage Examples
+
+***Request payload `PUT`- `/indexes/articles/settings/separator-tokens`***
+```json
+["|", "/", "&sep"]
```

Comment:
Isn't `"/"` a separator by default? Using this list as a reference: https://docs.rs/charabia/0.8.3/src/charabia/separators.rs.html#16-62

Reply:
Yes it is